packages/nextjs/pages/api/etl/index.ts

```diff
@@ -110,14 +112,14 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
               },
             });
 
-            for (const metric of Object.keys(metricNamesObj) as (keyof Metrics)[]) {
-              if (projectMetrics[metric]) {
-                globalScoreData[metric] += projectMetrics[metric];
-              }
-            }
+            // Add this projects impact_index score to the global score for the day
+            globalScoreData.impact_index += impact_index;
           }
 
-          globalScoreData.impact_index = getImpactIndex(globalScoreData, weightings, maxScoreByMetric);
+          // Adjust to get the average impact_index for the day
+          if (globalScoreData.impact_index && osoData.length) {
```

Comment:
In addition to how we are creating the impact index I would love to hear your thoughts on averaging the global impact index as I am doing here. To me it boils down to whether we want to show impact as growing when new projects enter the ecosystem. By averaging, we will only show when project impact is going up. I can see a case for both ways. I will ask Jonas as well.